pkg/build/archive.go

```diff
@@ -60,7 +60,7 @@ func Archive(manifest model.Manifest) error {
 
 		// Set up samples. We filter down to only some file patterns
 		// TODO - clean this up. We probably include files we don't want and exclude files we do want.
-		includePatterns = []string{"*.yaml", "*.md", "cleanup.sh", "*.txt", "*.pem", "*.conf", "*.tpl", "*.json", "Makefile"}
+		includePatterns = []string{"*.yaml", "*.md", "*.sh", "cleanup.sh", "*.txt", "*.pem", "*.conf", "*.tpl", "*.json", "Makefile"}
```

Comment:
Does *.sh cover cleanup.sh?

Reply:
Ahh, yes it includes the following files.
```
$ find . -type f -name "*.sh"
./bookinfo/build_push_update_images.sh
./bookinfo/platform/kube/cleanup.sh
./bookinfo/src/mongodb/script.sh
./bookinfo/src/build-services.sh
./certs/generate-workload.sh
./helloworld/src/build_service.sh
./helloworld/loadgen.sh
./rawvm/demo.sh
./multicluster/gen-eastwest-gateway.sh
```